src/bin/index.js

```diff
@@ -1,8 +1,18 @@
 #!/usr/bin/env node
 
+if (!process.env.DEBUG) {
+  process.env.DEBUG = "phenomic:*"
+}
+
 // still in src/, handy to play
 if (__filename.indexOf("phenomic/src") > 1) {
```

Comment:
https://ci.appveyor.com/project/MoOx/phenomic/build/1580#L2452

Path on windows is different from linux. ('/' vs '\')